jstz_api/tests/wptreport.json

```diff
@@ -0,0 +1,1176 @@
+{
+  "test_harness": {
+    "encoding": {
+      "Folder": {
+        "api-basics.any.js": {
+          "Test": {
+            "variations": [
+              {
+                "subtests": [
+                  {
+                    "name": "Default encodings",
+                    "status": "Fail",
+                    "message": "TextEncoder is not defined"
+                  },
+                  {
+                    "name": "Default inputs",
+                    "status": "Fail",
+                    "message": "TextEncoder is not defined"
+                  },
+                  {
+                    "name": "Encode/decode round trip: utf-8",
+                    "status": "Fail",
+                    "message": "TextEncoder is not defined"
+                  },
+                  {
+                    "name": "Decode sample: utf-16le",
+                    "status": "Fail",
+                    "message": "TextDecoder is not defined"
+                  },
+                  {
+                    "name": "Decode sample: utf-16be",
+                    "status": "Fail",
+                    "message": "TextDecoder is not defined"
+                  },
+                  {
+                    "name": "Decode sample: utf-16",
+                    "status": "Fail",
+                    "message": "TextDecoder is not defined"
+                  }
+                ],
+                "status": "Err",
+                "metrics": {
+                  "passed": 0,
+                  "failed": 6,
+                  "timed_out": 0
+                }
```

Comment:
question: What does it mean for the status to be "Err" here ?

Reply:
It means that the overall test failed (in this case bcs all the subtests failed)

Reply:
Uugh sorry I commented the wrong place.
Here is a better example from my branch [here](https://github.com/trilitech/jstz/blob/c49e6b3421c99acc72f99487fc8183f0b9f4c21f/jstz_api/tests/wptreport.json#L447-L477) for which all the subtests pass, but I still have an Err status

```
"textdecoder-byte-order-marks.any.js": {
          "Test": {
            "variations": [
              {
                "subtests": [
                  {
                    "name": "Byte-order marks: utf-8",
                    "status": "Pass",
                    "message": null
                  },
                  {
                    "name": "Byte-order marks: utf-16le",
                    "status": "Pass",
                    "message": null
                  },
                  {
                    "name": "Byte-order marks: utf-16be",
                    "status": "Pass",
                    "message": null
                  }
                ],
                "status": "Err",
                "metrics": {
                  "passed": 3,
                  "failed": 0,
                  "timed_out": 0
                }
              }
            ]
          }
        }
```

Reply:
🤔 https://github.com/trilitech/jstz/blob/c49e6b3421c99acc72f99487fc8183f0b9f4c21f/jstz_api/tests/wpt.rs#L321-L327

Reply:
In this case, it might be due to the fact that the test harness doesn't call the completion callback for some reason 🤔

Reply:
In this case, I will modify it to have a `Option`al status

Reply:
Added the `WptTestStatus::Null` variant